src/wp-includes/theme.php

```diff
@@ -4203,6 +4203,21 @@ function wp_is_block_theme() {
 	return wp_get_theme()->is_block_theme();
 }
 
+/**
+ * Given an element name, returns a class name.
+ * Alias of WP_Theme_JSON::get_element_class_name.
+ *
+ * @since 6.1.0
+ *
+ * @param string $element The name of the element.
+ *
+ * @return string The name of the class.
+ *
```

Comment:
```suggestion
```
Just need to remove the empty line at the end of the docblock. :slightly_smiling_face:

Reply:
Oops 😅